submissions/evgenii-del/html-css-popup/index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="./css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto" rel="stylesheet">
+    <meta charset="UTF-8">
+    <title>Kottans popup</title>
+</head>
+<body>
+<div class="wrapper">
+    <nav class="nav">
+        <div class="nav__items">
+            <div class="nav__item nav__link">
+                <a class="link" href="#">Mail</a>
+            </div>
+            <div class="nav__item nav__link">
+                <a class="link" href="#">Images</a>
+            </div>
+            <div class="nav__item nav__button">
+                <label for="popup_show"></label>
+                <input type="checkbox" class="popup__btn nav__img" id="popup_show">
```

Comment:
The classes are not entirely logical here. This is where input is `popup__btn` and at the same time it is `nav__img`. This naming confusion is better to avoid in the future.